contentScript.js

```diff
@@ -12,7 +12,18 @@ const isOnJoinPage = () =>
 const hideSelfButton = () =>
   document
     .evaluate(
-      "//div[following-sibling::span//i[contains(., 'close_fullscreen')]]",
+      "//span[i[contains(., 'close_fullscreen')]]",
+      document,
+      null,
+      XPathResult.ANY_TYPE,
+      null
+    )
+    .iterateNext();
+
+const moreButton = () =>
+  document
+    .evaluate(
+      "//button[i[contains(., 'more_vert')]]",
       document,
       null,
       XPathResult.ANY_TYPE,
```

Comment:
I believe this will need to be more specific to target the "More" button on your own tile, as this currently just finds the first one (which might be someone else)

Reply:
Perhaps we can match on the user's name and use that to find the right box but that seems a bit fragile 🤔